linux/scripts/reconf.sh

```diff
@@ -54,6 +54,7 @@ done
 for proj in ${extra_projects}; do
     if [ "${proj}" == "keyboardprocessor" ]; then
         rm -rf keyboardprocessor
+        cp ../VERSION.md ../common/core/desktop/
```

Comment:
Can we add the VERSION.md in common/core/desktop into .gitignore?

Reply:
Good point. Done.